ci/run.bat

```diff
@@ -3,7 +3,7 @@ set AGORA_VERSION="HEAD"
 rem call dub test --skip-registry=all --compiler=%DC%
 call dub build --skip-registry=all --compiler=%DC% -c unittest -b unittest
 if %errorlevel% neq 0 exit /b %errorlevel%
-call rdmd --build-only --compiler=%DC% ./tests/runner.d --compiler=%DC% -cov
+call %DC% ./tests/runner.d %DC% -cov
```

Comment:
Isn't there the option "-i -run" in Windows environment?

Reply:
Correct

Reply:
So actually, `-i` was missing, but `-run` shouldn't be used because we were not running those tests...